optuna/integration/botorch.py

```diff
@@ -652,7 +652,7 @@ def sample_independent(
     def reseed_rng(self) -> None:
         self._independent_sampler.reseed_rng()
         if self._seed is not None:
-            self._seed = numpy.random.RandomState().randint(2**60)
+            self._seed = numpy.random.RandomState().randint(2**60, dtype=numpy.int64)
```

Comment:
As discussed offline, let us set the seed within int32 range instead of specifying dtype.
```suggestion
            self._seed = numpy.random.RandomState().randint(np.iinfo(np.int32).max)
```